cocotb/share/lib/gpi/GpiCommon.cpp

```diff
@@ -182,7 +187,7 @@ static void gpi_load_libs(std::vector<std::string> to_load)
             func_name = std::string(it+1, arg.end());
         }
 
-        std::string full_name = "lib" + lib_name + DOT_LIB_EXT;
+        std::string full_name = LIB_PREFIX_STR + lib_name + DOT_LIB_EXT;
```

Comment:
I was never a fan of this. I'm not sure why the user shouldn't just pass the full name. This prevents the ability to pass relative or absolute paths. I'll split that out into another issue.